diff --git a/lib/currency.c b/lib/currency.c
index d02dfc4..0b15311 100644
--- a/lib/currency.c
+++ b/lib/currency.c
@@ -540,7 +540,7 @@ CurrencyManager* currency_manager_get_default (void) {
 	_tmp119_ = currency_new ("ZAR", _ ("South African Rand"), "R");
 	_tmp118_->priv->currencies = g_list_append (_tmp118_->priv->currencies, _tmp119_);
 	_tmp120_ = default_currency_manager;
-	currency_manager_download_rates (_tmp120_);
+//	currency_manager_download_rates (_tmp120_);
 	_tmp121_ = default_currency_manager;
 	_tmp122_ = _g_object_ref0 (_tmp121_);
 	result = _tmp122_;
