diff --git a/src/gnome-calculator.c b/src/gnome-calculator.c
index 19ed32b..80bfc62 100644
--- a/src/gnome-calculator.c
+++ b/src/gnome-calculator.c
@@ -1590,7 +1590,6 @@ gint calculator_main (gchar** args, int args_length1) {
 	_tmp4_ = g_path_get_basename (_tmp3_);
 	_g_free0 (calculator_program_name);
 	calculator_program_name = _tmp4_;
-	gtk_window_set_default_icon_name ("accessories-calculator");
 	_tmp5_ = calculator_new ();
 	app = _tmp5_;
 	_tmp6_ = args;
